src/Models/ResponseModels/SDGResponse.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace SolutionsService.Models.ResponseModels
+{
+    public class SDGResponse
+    {
+        public Guid Id { get; set; }
+        public int SDGNumber { get; set; }
```

Comment:
Shouldn't this be an enum? Potentially a string?

Reply:
Enums cannot be content managed easily. The Goals could be reviewed and reworded annually

Reply:
In this case, there's also the danger they can be renumbered

Reply:
It's not very easy for a user, like an admin, to add a whole new value to an enum rather than storing the list of SDGs into a database that can be added to and updated.